### DISCUSSION TITLE
hack-browser-data-0.4.6

### DISCUSSION THREAD

**Discussion by moonD4rk** [moonD4rk/HackBrowserData#364] [Announcements]
## What's Changed
* dev: refactor items and update repo deploy by @moonD4rk in https://github.com/moonD4rk/HackBrowserData/pull/278
* feat: refactor logger to standard library by @moonD4rk in https://github.com/moonD4rk/HackBrowserData/pull/280
* dev: Remove CGO go-sqlite3 with pure go driver by @moonD4rk in https://github.com/moonD4rk/HackBrowserData/pull/292
* dev: Optimize encryption and decryption module by @moonD4rk in https://github.com/moonD4rk/HackBrowserData/pull/305
* refactor: refactor the logic for obtaining browser data. by @moonD4rk in https://github.com/moonD4rk/HackBrowserData/pull/321
* fix: skip chromium snapshot dir to find right password database by @moonD4rk in https://github.com/moonD4rk/HackBrowserData/pull/353
* fix: Improve error handling if walk browser profile dir by @moonD4rk in https://github.com/moonD4rk/HackBrowserData/pull/354
* chore: Optimize GitHub actions and dependencies updates by @moonD4rk in https://github.com/moonD4rk/HackBrowserData/pull/362

**Full Changelog**: https://github.com/moonD4rk/HackBrowserData/compare/v0.4.5...v0.4.6

<hr /><em>This discussion was created from the release <a href='https://github.com/moonD4rk/HackBrowserData/releases/tag/v0.4.6'>hack-browser-data-0.4.6</a>.</em>

**Comment by uaoao12** [comment]
请问 那个版本支持 win7呢？谢谢。

**Comment by salmanmalik22110-boop** [comment]
Subway surf hack
